roles/galaxy_post_install/tasks/setup_galaxy_api_access_log.yml

```diff
@@ -0,0 +1,40 @@
+---
+- block:
+  - name: Create Galaxy api access log file
+    file:
+      path: '/var/log/galaxy_api_access.log'
+      state: touch
+      owner: '{{ pulp_user }}'
+      group: '{{ pulp_group }}'
+      mode: "u+rw"
+
+  - name: Verify /etc/default/pulpcore-api exists
+    stat:
+      path: /etc/default/pulpcore-api
+    register: stat_result
+
+  - name: Create /etc/default/pulpcore-api
+    file:
+      path: /etc/default/pulpcore-api
+      state: touch
+      owner: '{{ pulp_user }}'
+      group: '{{ pulp_group }}'
+      mode: "u+rw"
+    when: not stat_result.stat.exists
+
+  - name: Verify /etc/default/pulpcore-api exists
+    stat:
+      path: /etc/default/pulpcore-api
+    register: stat_result2
+
+  - name: Set environment variable for Galaxy api access logging
+    lineinfile:
+      path: /etc/default/pulpcore-api
+      line: GALAXY_ENABLE_API_ACCESS_LOG={{ pulp_settings['galaxy_enable_api_access_log'] | default(False) }}
+      insertafter: EOF
+      state: present
+    notify: Restart pulpcore-api.service
+    when:
+      - stat_result2.stat.exists
```

Comment:
shouldn't it raise an error when
```yaml
not stat_result2.stat.exists
```

Reply:
@fao89 good point.  I think that should be addressed now.  thanks!